src/pages/components/MoviesList/MoviesList.js

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState } from 'react';
+import MovieCard from '../MovieCard/MovieCard';
+import { Grid, Typography, Pagination, FormControl, InputLabel, Select, MenuItem } from '@mui/material';
+import Header from '../Header/Header';
+import styles from './MoviesList.module.css';
+
+function MovieList() {
+  const [movies, setMovies] = useState([]);
+  const [page, setPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+  const [sortOption, setSortOption] = useState('all');
+  const [searchValue, setSearchValue] = useState("");
+  let [genres, setGenres] = useState([]);
+  let [genre, setGenre] = useState("");
+
+  useEffect(() => {
+    const fetchMovies = async (page, sortOption, searchValue) => {
+      console.log(process.env.NEXT_PUBLIC_API_URL_MOVIES)
+      // const response = await fetch(`http://127.0.0.1:3001/movies?page=${page}&sort=${sortOption}&search=${searchValue}&genreId=${genre}`);
```

Comment:
please remove the commented code if it is not used. especially if it includes variables that are present in the .env